### PR TITLE
App monitoring + Open app + App authentication

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**

### CODE REVIEW

File: content/md/connectivity/manage-your-apps.md [hunk 1]
```diff
@@ -11,75 +11,165 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
-To access the settings of an App, click on `Manage App` and you will access the first page of the app settings, then click on the `Permissions` tab.
+# Open a connected app interface
+To open a connected app interface, click on `Open app` in the `Apps` menu. 
+The app interface will opens in a new tab of your navigator. 
 
-This is where you can view and update permissions granted to an App.
+![Open app button](../img/app-card-open.png) 
 
-![Connected app permissions](../img/connected-app-permissions.png) <!--Update screenshot-->
+When you open an app, Akeneo PIM may ask you to agree to share some of your user information to directly authenticate your on the app. To do so, click on `Agree` and you will be automaticaly redirected to the app.
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-apps.md:51]:
```suggestion
When you open an app, Akeneo PIM may ask you to agree to share some of your user information to directly authenticate you on the app. To do so, click on `Agree` and you will be automatically redirected to the app.
```

File: content/md/connectivity/manage-your-apps.md [hunk 2]
```diff
@@ -11,75 +11,165 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+By default, all connected apps have the `Other` flow type, so you won't have any app monitoring for your connected apps unless you update their flow type by choosing the `Data source` or the `Data destination` option.
+
+### The `Data source` flow type
+
+Choose this option for your app whenever it represents a data flow entering the PIM. For example, select this option if the connected app mainly creates or updates PIM data, such as products, product models, assets, or reference entities.
+
+If you choose this option, the Data flows dashboard will focus on this app's data pushed **inside** the PIM.
+
+::: info
+With this flow type, you can connect ERP, DAM, MDM,...
```
Comment (meganelepalud-akeneo) [inline content/md/connectivity/manage-your-apps.md:88]:
```suggestion
With this flow type, you can connect ERP, DAM, or MDM.
```

File: content/md/connectivity/manage-your-apps.md [hunk 3]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
 **THIS FEATURE WILL BE AVAILABLE SOON**
 :::
 
-# Manage your connected Apps
-
 Now that you connected one or several Apps with your Akeneo PIM, you probably want to see the list of all the connected Apps and access their settings.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:14]:
```suggestion
Now that you connected one or several Apps with your Akeneo PIM, you probably want to see them listed in the UI and access their settings.
```

File: content/md/connectivity/manage-your-apps.md [hunk 4]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
 
-This is the role of the `Connect` sub-menu called `Apps`. Here you can see information about connected Apps. Their name, logo, category, and who owns it.
+This is the role of the `Connect` sub-menu called `Apps`. Here you can see information about connected Apps. Their name, logo, category, and who owns it.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:16]:
```suggestion
This is the role of the `Connect` sub-menu called `Apps`. Here you can see information about connected Apps: their name, logo, category, and who owns it.
```

File: content/md/connectivity/manage-your-apps.md [hunk 5]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
-This is where you can see which authorizations have been granted during the activation process.
+To give you the capacity to choose who can manage or open an App, we added two new Role permissions in the PIM. 
 
-![Connected app Settings](../img/connected-app-settings.png)
+To manage an App, at least one of your user roles needs the `Manage apps` permission, and to open a connected App interface, at least one of your user roles needs the `Open apps` permission.
 
+To give role permission: 
+- Go to `System`, then `Roles`
+- Click on the role you want to update
+- Go to the `Permissions` tab and select the System entry
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:36]:
```suggestion
- Go to `Permissions` and select the System entry
```

File: content/md/connectivity/manage-your-apps.md [hunk 6]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
 
 ::: warning
-This tab is only available for EE users.
+If you can't update user roles, please ask your administrator. 
 :::
 
-To access the settings of an App, click on `Manage App` and you will access the first page of the app settings, then click on the `Permissions` tab.
+# Open a connected App interface
+To open a connected App interface, click on `Open app` in the `Apps` menu. 
+The App interface will opens in a new tab of your navigator.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:47]:
```suggestion
The App interface will open in a new tab of your navigator. 
```

File: content/md/connectivity/manage-your-apps.md [hunk 7]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
 
-If you want to know more about App permissions, please read this paragraph of our documentation: [How to give permissions to your App?](how-to-connect-my-pim-with-apps.html#how-to-give-permissions-to-your-app-ee-only) 
+![Wizard authentication](../img/wizard-authentication.png) 
 
-## Who can manage App permissions?
+ 
+# Manage your connected App settings
 
-To give the capacity to choose who can see the `Apps` menu and manage an App, we added a new Role permission in the PIM: `Manage Apps`. To manage an App, your user role needs it.
+To access a connected App's settings, click on `Manage app` in the `Apps` menu.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:58]:
```suggestion
To access the connected App settings, click on `Manage app` in the `Apps` menu. 
```

File: content/md/connectivity/manage-your-apps.md [hunk 8]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+To access a connected App's settings, click on `Manage app` in the `Apps` menu. 
 
-To give the role permission: 
-- Go to `System`, then `Roles`
-- Click on the role you want to update
-- Go to the `Permissions` tab and select the System entry
-- Finally enable the `Manage apps` option.
+![Manage App button](../img/app-card-manage.png) 
+
+Each connected App has 3 tabs:
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:62]:
```suggestion
You will see 3 tabs in that section: 
```

File: content/md/connectivity/manage-your-apps.md [hunk 9]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+The flow type allows you to characterize the data flows interacting with your PIM. More precisely, it allows you to indicate the **direction** of a given flow.
+
+It will determine how your App flows are monitored in the [data flows dashboard](connection-dashboard.html).
+
+This flow type has three available options described in the following sections. It will help you make your decision.
+
+### The `Other` flow type
+
+By default, all connected Apps have the `Other` flow type, so you won't have any App monitoring for your connected Apps unless you update their flow type by choosing the `Data source` or the `Data destination` option.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:79]:
```suggestion
The `Other` flow type is the default value for all connected Apps, so there is no App monitoring for your connected Apps. To enable App monitoring, make sure you update their flow type by choosing the `Data source` or the `Data destination` option.
```

File: content/md/connectivity/manage-your-apps.md [hunk 10]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+### The `Data source` flow type
+
+Choose this option for your App whenever it represents a data flow entering the PIM. For example, select this option if the connected App mainly creates or updates PIM data, such as products, product models, assets, or reference entities.
+
+If you choose this option, the Data flows dashboard will focus on this App's data pushed **inside** the PIM.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:85]:
```suggestion
If you choose this option, the Data flows dashboard will focus on this App's data pushed **into** the PIM.
```

File: content/md/connectivity/manage-your-apps.md [hunk 11]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+::: info
+With this flow type, you can connect ERP, DAM, or MDM.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:88]:
```suggestion
With this flow type, you can connect your ERP, DAM, or MDM.
```

File: content/md/connectivity/manage-your-apps.md [hunk 12]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+:::
+
+### The `Data destination` flow type
+
+Choose this option for your App to represent a data flow that mainly extracts product information from your PIM.
+
+If you choose this option, the Data flows dashboard will focus on this App's data pushed **outside** the PIM.
+
+::: info
+You can connect e-commerce, publishing, or syndication Apps with this flow type.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:98]:
```suggestion
You can connect your e-commerce, publishing, or syndication Apps with this flow type.
```

File: content/md/connectivity/manage-your-apps.md [hunk 13]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+:::
+
+## Enable the tracking
+
+You can choose to track a connected App in [Data flows](connection-dashboard.html).
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:103]:
```suggestion
You can choose to monitor a connected App in [Data flows](connection-dashboard.html).  
```

File: content/md/connectivity/manage-your-apps.md [hunk 14]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+Choose this option for your App to represent a data flow that mainly extracts product information from your PIM.
+
+If you choose this option, the Data flows dashboard will focus on this App's data pushed **outside** the PIM.
+
+::: info
+You can connect e-commerce, publishing, or syndication Apps with this flow type.
+:::
+
+## Enable the tracking
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:101]:
```suggestion
## Enable Apps monitoring
```

File: content/md/connectivity/manage-your-apps.md [hunk 15]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+You can choose to track a connected App in [Data flows](connection-dashboard.html).  
+To do so, check `Track in Data flows dashboard` in the setting screen of the connected App you want to track.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:104]:
```suggestion
To do so, check `Track in Data flows dashboard` in the settings section of the connected App you want to monitor.
```

File: content/md/connectivity/manage-your-apps.md [hunk 16]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+<img class="img-responsive in-article" alt="Track checkbox" src="../img/track-checkbox.png" style="max-width: 250px;">
+
+::: warning
+We don't track Apps with the `Others` flow type.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:109]:
```suggestion
We don't monitor Apps with the `Other` flow type.  
```

File: content/md/connectivity/manage-your-apps.md [hunk 17]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+## Enable the tracking
+
+You can choose to track a connected App in [Data flows](connection-dashboard.html).  
+To do so, check `Track in Data flows dashboard` in the setting screen of the connected App you want to track.
+
+<img class="img-responsive in-article" alt="Track checkbox" src="../img/track-checkbox.png" style="max-width: 250px;">
+
+::: warning
+We don't track Apps with the `Others` flow type.  
+If you want to track the data entering your PIM, [change your connected App flow type](#choose-your-flow-type) to `Data source`.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:110]:
```suggestion
If you want to monitor the data entering your PIM, [change your connected App flow type](#choose-your-flow-type) to `Data source`.  
```

File: content/md/connectivity/manage-your-apps.md [hunk 18]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+You can choose to track a connected App in [Data flows](connection-dashboard.html).  
+To do so, check `Track in Data flows dashboard` in the setting screen of the connected App you want to track.
+
+<img class="img-responsive in-article" alt="Track checkbox" src="../img/track-checkbox.png" style="max-width: 250px;">
+
+::: warning
+We don't track Apps with the `Others` flow type.  
+If you want to track the data entering your PIM, [change your connected App flow type](#choose-your-flow-type) to `Data source`.  
+If you want to track the data pulled out of the PIM by your App, [change your connected App flow type](#choose-your-flow-type) to `Data destination`.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:111]:
```suggestion
If you want to monitor the data pulled out of the PIM by your App, [change your connected App flow type](#choose-your-flow-type) to `Data destination`.
```

File: content/md/connectivity/manage-your-apps.md [hunk 19]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+:::
+
+::: info
+You can enable or disable the tracking whenever you want.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:115]:
```suggestion
You can enable or disable the monitoring whenever you want.  
```

File: content/md/connectivity/manage-your-apps.md [hunk 20]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+::: warning
+We don't track Apps with the `Others` flow type.  
+If you want to track the data entering your PIM, [change your connected App flow type](#choose-your-flow-type) to `Data source`.  
+If you want to track the data pulled out of the PIM by your App, [change your connected App flow type](#choose-your-flow-type) to `Data destination`.
+:::
+
+::: info
+You can enable or disable the tracking whenever you want.  
+We will stop tracking the connected App immediately, and you won't be able to select it in the filters of the Data flows dashboard. However, note that all the data collected up to that point for this connected App will still appear in some KPIs.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:116]:
```suggestion
We will stop monitoring the connected App immediately, and you won't be able to select it in the filters of the Data flows dashboard. However, note that all the data collected up to that point for this connected App will still appear in some KPIs.
```

File: content/md/connectivity/manage-your-apps.md [hunk 21]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+:::
+
+
+## Consult authorizations granted to your App
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:120]:
```suggestion
## Access authorizations granted to your App
```

File: content/md/connectivity/manage-your-apps.md [hunk 22]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+To access an App's settings, click on `Manage app`, and you will access the first page of the App settings.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:122]:
```suggestion
To access an App's settings, click on `Manage app`. 
```

File: content/md/connectivity/manage-your-apps.md [hunk 23]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+You can enable or disable the tracking whenever you want.  
+We will stop tracking the connected App immediately, and you won't be able to select it in the filters of the Data flows dashboard. However, note that all the data collected up to that point for this connected App will still appear in some KPIs.
+:::
+
+
+## Consult authorizations granted to your App
+
+To access an App's settings, click on `Manage app`, and you will access the first page of the App settings. 
+
+This page is where you can see which authorizations have been granted during the activation process.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:124]:
```suggestion
Here is where you can see which authorizations have been granted during the activation process.
```

File: content/md/connectivity/manage-your-apps.md [hunk 24]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+![Connected App Settings](../img/connected-app-settings.png)
+
+
+## Setup your App permissions `EE`
+
+::: warning
+This tab is only available for EE users.
+:::
+
+To access an App's settings, click on `Manage app` and you will access the first page of the App settings, then click on the `Permissions` tab.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:135]:
```suggestion
To access an App's settings, click on `Manage app` and you will access the first page of the App settings, then click on the `Permissions` tab.
```
```suggestion
To access an App's settings, click on `Manage app`, then click on `Permissions`.
```

File: content/md/connectivity/manage-your-apps.md [hunk 25]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+This page allows you to view and update permissions granted to an App.
+
+![Connected App permissions](../img/connected-app-permissions.png) 
+
+::: info
+If you want to know more about App permissions, please read this paragraph of our documentation: [How to give permissions to your App?](how-to-connect-my-pim-with-apps.html#how-to-give-permissions-to-your-app-ee-only)
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:142]:
```suggestion
If you want to know more about App permissions, please read this section of our documentation: [How to give permissions to your App?](how-to-connect-my-pim-with-apps.html#how-to-give-permissions-to-your-app-ee-only) 
```

File: content/md/connectivity/manage-your-apps.md [hunk 26]
```diff
@@ -11,56 +11,146 @@ related: how-to-connect-my-pim-with-apps, connection-dashboard
...
+
+::: info
+If you want to know more about App permissions, please read this paragraph of our documentation: [How to give permissions to your App?](how-to-connect-my-pim-with-apps.html#how-to-give-permissions-to-your-app-ee-only) 
+:::
 
-![Role manage apps](../img/role-manage-apps.png)
+### Who can manage App permissions?
 
-The PIM is also using the following role permissions to manage who can grant permission to an App: 
+The PIM is using the following role permissions to manage who can grant permission to an App:
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-apps.md:147]:
```suggestion
The PIM uses the following role permissions to manage who can grant permission to an App: 
```

File: content/md/connectivity/manage-your-connections.md [hunk 27]
```diff
@@ -66,7 +66,7 @@ This flow type has three available options you'll have to choose from. The next
 
 ## The `Source connection` flow type
 
-Choose this option for you connection whenever it represents a data flow entering the PIM. For example, your data flow is mainly creating or updating PIM data, such as products, product models, assets, reference entities,...
+Choose this option for your connection whenever it represents a data flow entering the PIM. For example, your data flow is mainly creating or updating PIM data, such as products, product models, assets, reference entities,...
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:69]:
```suggestion
Choose this option for your connection whenever it represents a data flow entering the PIM. For example, your data flow mainly creates or updates PIM data, such as products, product models, assets, reference entities,...
```

File: content/md/connectivity/manage-your-connections.md [hunk 28]
```diff
@@ -66,7 +66,7 @@ This flow type has three available options you'll have to choose from. The next
...
 
 ::: info
 With this flow type, you can connect ERP, DAM, MDM,...
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:72]:
```suggestion
With this flow type, you can connect your ERP, DAM, MDM, etc.
```

File: content/md/connectivity/manage-your-connections.md [hunk 29]
```diff
@@ -76,7 +76,7 @@ If you choose this option, the Data flows dashboard will focus on the data that
 
 ## The `Destination connection` flow type
 
-Choose this option for you connection whenever it represents a data flow that is mainly extracting product information from your PIM.
+Choose this option for your connection whenever it represents a data flow that is mainly extracting product information from your PIM.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:79]:
```suggestion
Choose this option for your connection whenever it represents a data flow that mainly extracts product information from your PIM.
```

File: content/md/connectivity/manage-your-connections.md [hunk 30]
```diff
@@ -100,7 +100,7 @@ That being said, even in this case, we advise you to choose between the two opti
 # Enable the tracking
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:100]:
```suggestion
# Enable the monitoring
```

File: content/md/connectivity/manage-your-connections.md [hunk 31]
```diff
@@ -100,7 +100,7 @@ That being said, even in this case, we advise you to choose between the two opti
...
 
 For each connection, you can choose whether you want to track it in [Data flows](connection-dashboard.html).
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:102]:
```suggestion
For each connection, you can choose whether you want to monitor it in [Data flows](connection-dashboard.html).  
```

File: content/md/connectivity/manage-your-connections.md [hunk 32]
```diff
@@ -100,7 +100,7 @@ That being said, even in this case, we advise you to choose between the two opti
 # Enable the tracking
 
 For each connection, you can choose whether you want to track it in [Data flows](connection-dashboard.html).  
-To do so, check `Track this connection in the dashboard`, in the connection setting screen of the connection you want to track.
+To do so, check `Track in Data flows dashboard`, in the connection setting screen of the connection you want to track.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:103]:
```suggestion
To do so, check `Track in Data flows dashboard`, in the connection settings screen of the connection you want to track.
```

File: content/md/connectivity/manage-your-connections.md [hunk 33]
```diff
@@ -112,7 +112,7 @@ If you want to track the data that is pulled out of the PIM by your connector, [
 
 ::: info
 You can deactivate the tracking whenever you want.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:114]:
```suggestion
You can disable the monitoring whenever you want.  
```

File: content/md/connectivity/manage-your-connections.md [hunk 34]
```diff
@@ -112,7 +112,7 @@ If you want to track the data that is pulled out of the PIM by your connector, [
 
 ::: info
 You can deactivate the tracking whenever you want.  
-We will stop tracking the connection immediately and you won't be able to select it in the filters of the connection dashboard. However, note that all the data collected up to that point for this connection will still appear in some KPIs.
+We will stop tracking the connection immediately and you won't be able to select it in the filters of the Data flows dashboard. However, note that all the data collected up to that point for this connection will still appear in some KPIs.
```
Comment (camillebourjade) [inline content/md/connectivity/manage-your-connections.md:115]:
```suggestion
We will stop monitoring the connection immediately and you won't be able to select it in the filters of the Data flows dashboard. However, note that all the data collected up to that point for this connection will still appear in some KPIs.
```

**Review by camillebourjade — APPROVED** [review]
English Review

**Pull request merged**